step-templates/releasenoteshub-pull-trigger.json

```diff
@@ -0,0 +1,83 @@
+  "ActionType": "Octopus.Script",
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Packages": [],
+  "Properties": {
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "Octopus.Action.Script.ScriptBody": "[system.version]$v \n\nif ( -not [string]::IsNullOrEmpty( $Version ) )\n{\n\t$v = New-Object System.Version($Version)\n}\nelse\n{\n\t$v = New-Object System.Version($OctopusParameters['Octopus.Release.Number'])\n}\n\n$headers = @{\n    \"Authorization\" = \"ApiKey $ApiKey\"\n}\n\n$body = @{\n    \"version\" = @{\"VersionMajor\" = $v.Major; \"VersionMinor\" = $v.Minor; \"VersionBuild\" = $v.Build; \"VersionRevision\" = $v.Revision};\n    \"createOnNotFound\" = $CreateOnNotFound;\n    \"merge\" = $Merge; \n    \"publish\" = $Publish     \n} | ConvertTo-Json\n\ntry {\n     Invoke-RestMethod -Method Post -Uri \"https://api.releasenoteshub.com/api/pull/PullVersion/$ProjectId\" -Headers $headers -Body $body -ContentType application/json-patch+json\n} catch {\n    Write-Host \"StatusCode:\" $_.Exception.Response.StatusCode.value__ \n    Write-Host \"StatusDescription:\" $_.Exception.Response.StatusDescription\n    throw $_.Exception\n}"
+  },
+  "Parameters": [
+    {
+      "Id": "3341141f-ab5d-4204-9d84-b46cb0e1ba71",
+      "Name": "ApiKey",
+      "Label": "ApiKey",
+      "HelpText": "https://support.releasenoteshub.com/article/show/105050-how-do-i-get-an-api-key",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      }
+    },
+    {
+      "Id": "02e58f68-5a86-41cd-9b21-636d5eb162bf",
+      "Name": "Version",
+      "Label": "Version",
+      "HelpText": "This is an optional parameter. If left blank, the Octopus Deploy system variable 'Octopus.Release.Number' will be used.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "0d3f067f-1432-4b3a-9c63-9a2eb9b6b522",
+      "Name": "ProjectId",
+      "Label": "RnHub Project Id",
+      "HelpText": "https://support.releasenoteshub.com/article/show/111560-how-to-get-project-id",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "8218468f-36f4-48d3-8e93-4a04745b6bac",
+      "Name": "Publish",
+      "Label": "Publish",
+      "HelpText": null,
+      "DefaultValue": "false",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Checkbox"
+      }
+    },
+    {
+      "Id": "6fa58ea9-b392-4f16-ab8c-07096e53a130",
+      "Name": "Merge",
+      "Label": "Merge",
+      "HelpText": "https://support.releasenoteshub.com/article/show/108944-what-is-a-merge-in-rnhub",
```

Comment:
Could you include a one-liner to help what this parameter is, and include the link for when people need further info- markdown is supported.
(This would allow users to only go to another place from Octopus if they still need further info)